students/seungho-shin/user/views.py

```diff
@@ -0,0 +1,34 @@
+import json
+
+from django.http            import JsonResponse
+from django.views           import View
+from django.core.exceptions import ValidationError
+from user.models            import User
+from user.validators        import validate_email
+
+class SignupView(View):
+    def post(self, request):
+        data    = json.loads(request.body)
+        try:
+            if not validate_email(data['email']):
+                return JsonResponse({'MASSAGE':'INVALID EMAIL'}, status=400)    
+            
+            required_pw_digits = 8
```

Comment:
파이썬에서 상수는 대문자로만 이루어져 있습니다.
```suggestion
            REQUIRED_LENGTH = 8
```